src/test/java/ttt/ConsoleBoardTest.java

```diff
@@ -6,6 +6,9 @@
 
 public class ConsoleBoardTest {
 
+    Marks X = Marks.X;
+    Marks O = Marks.O;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/mollie-c-stephenson/TicTacToe/file/2583160145/issues/source?bid=3284885&fileBranchId=3303498#l10)